### ISSUE TITLE
Clarify the validity of the recorded timing values for request started and first byte

### ISSUE THREAD

**Issue by jcmorrow** [RecordReplay/devtools#4806]
Right now it looks like there are discrepancies between what Chrome reports for request timings and what we are reporting. It's a little tough to tell because we have slightly different sets of points (for instance, I believe right now we *only* get time initiated and first-byte). By the end of this we should know: 

- how often we are out of sync with Chrome's reporting
- by how much
- how to best translate the numbers we have into what devs are familiar with from Chrome

**Comment by jcmorrow** [comment]
We seem pretty consistently off from what the Replay browser itself records. It's sometimes difficult to line up because we don't have response sizes (which are useful for figuring out which request is which when there are several to the same endpoint), but we are consistently reporting much higher times to first byte than the Replay browser or Chrome reports for the entire request. For instance, even when things are coming from cache we record a ~30ms response time, but Replay and Chrome both report 0ms (I'm not sure if that is hardcoded or not, I did at some point see `-1ms` in the Replay browser, which makes me think they are *not* hardcoded). I'm not sure why this is happening, but I think we should probably make it a goal some time to dive into these differences more specifically and understand what times Chrome/Replay might be recording that we are not.

**Comment by jcmorrow** [comment]
Actually, I have no idea what I was seeing before. I've set up a small local server that takes requests in the from of `http://localhost:3000/N` where N is a number and returns 200 OK in N milliseconds, and then recorded this page:

```
<html>
  <body>
    <script type="text/javascript">
      setTimeout(() => {
        fetch("http://localhost:3000/0")
      }, 100);
      setTimeout(() => {

        fetch("http://localhost:3000/100")
      }, 500);
      setTimeout(() => {
        fetch("http://localhost:3000/200")
      }, 1000);
      setTimeout(() => {
        fetch("http://localhost:3000/1000")
      }, 2000);
      setTimeout(() => {
        fetch("http://localhost:3000/5000")
      }, 3000);
    </script>
  </body>
</html>
```
And I get the following timings:

```
localhost:3000/0: Start: 572, End: 580, Time: 8
localhost:3000/100: Start: 975, End: 1082, Time: 107
localhost:3000/200: Start: 1478, End: 1687, Time: 209
localhost:3000/1000: Start: 2473, End: 3482, Time: 1009
localhost:3000/2000: Start: 3473, End: 8483, Time: 5010
```
These seem basically perfect? I think it's time to ship a super basic timing tab!

**Comment by jcmorrow** [comment]
Implemented in #5074
